### ISSUE TITLE
Delete files in file browser

### ISSUE THREAD

**Issue by MoritzWeber0** [DSD-DBS/capella-collab-manager#1287]
As alternative to complex file deletion workflows via session containers, it shall be possible to delete single files and empty directories from the file browser. To prevent unintended deletions, it shall not be possible to delete directories, which contain files.

**Comment by jamilraichouni** [comment]
> To prevent unintended deletions, it shall not be possible to delete directories, which contain files.

Dislike this. What about an extra question dialog like "Do you really want to delete the directory '/path/to/dir' which contains x file(s)?"

**Comment by MoritzWeber0** [comment]
> > To prevent unintended deletions, it shall not be possible to delete directories, which contain files.
> 
> For cases of non-empty directories I'd clearly prefer an extra question dialog like "Do you really want to delete the directory '/path/to/dir' which contains x file(s)?" instead of being forced to delete file by file.
> 
> Image a user exported all representations into a directory and would be forced to delete hundreds of diagrams.

Yes, that makes sense. A dialog can also prevent unintended deletions and is less annoying than deletion of a ton of files manually.

**Comment by Wuestengecko** [comment]
This would be really nice to have, and has now been requested a couple times by our users in one way or another.

Additionally, since neither the default Capella nor Jupyter images expose the entire `/workspace` directory (both only show subdirectories), with only those tools available the only way to delete files sitting directly below `/workspace` is to use bash within a Jupyter session, which is unreasonable to ask of end users.
